styleguide/source/assets/scss/01-atoms/_search-field.scss

```diff
@@ -17,6 +17,7 @@
     color: $gray-50;
     line-height: initial;
     padding: 1em 28px 3px 0;
+    background: transparent;
```

Comment:
I believe this has removed the white background on the global header search box.

Reply:
![image](https://user-images.githubusercontent.com/4438120/44609283-a2b10300-a7bc-11e8-9684-38f43816eaa7.png)